modules/creating-kubefed-CRs.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * kubefed/installing-kubefed.adoc
+
+[id="creating-kubefed-CRs_{context}"]
+= Creating KubeFed custom resources
+
+You must create the `KubeFedWebHook` and `KubeFed` custom resources (CRs) for {KubeFedProductShortName} deployment.
+The {KubeFedProductName} operator will create the required custom resource definitions (CRDs) for these CRs automatically.
+
+.Prerequisites
+* You will need to have the `oc` CLI tool installed.
+
+.Procedure
+
+. Create a `KubeFedWebHook` resource to instantiate an admission webhook controller for KubeFed. It is recommended to create this CR with `scope: Cluster` to allow the webhook controller to serve API requests coming from all KubeFed control planes within the same cluster.
```

Comment:
@abhat we should discuss whether the `KubeFedWebhook` resource should have a `scope` field, but this is factually accurate as it is today.

I would not phrase this as "recommended"; it's required. You could just change "recommended" to "required" here.